docs/GettingStarted.rst

```diff
@@ -1661,17 +1661,27 @@ the link.
 Posting in the activity stream via the New Composer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~	
 
-A new way to share in the activity stream is available via a clickable text "Post something to your network" from the home page and "Post something to '$Space Name' " from space's home page.
+A new way to share in the activity stream is available via a clickable text "Post" from the home page and "Post in '$Space Name' " from space's home page.
```

Comment:
I don't see any changes for this label in the code.

Reply:
these changes are not done (the tasks have been created today) but i changed the text to prevent this